static/js/unread_ops.js

```diff
@@ -111,6 +113,8 @@ export function process_unread_messages_event({message_ids, message_details}) {
                 stream_id: message_info.stream_id,
                 topic: message_info.topic,
             });
+
+            inbox_ui.update_unread_count(message_info.stream_id, message_info.topic);
         }
```

Comment:
Hmm, reading this code, don't we need to update something when a private message gets marked as unread too?

@amanagr is this an existing bug in recent topics? I don't see a parallel code path there either...